boards/adi/max32672evkit/max32672evkit.dts

```diff
@@ -50,6 +52,42 @@
 		sw0 = &pb1;
 		watchdog0 = &wdt0;
 	};
+
+	mipi_dbi {
+		compatible = "zephyr,mipi-dbi-spi";
+		/* Enable D/C line for 4wire mode */
+		/* dc-gpios = <&gpio0 19 (GPIO_ACTIVE_HIGH | MAX32_GPIO_VSEL_VDDIOH)>; */
```

Comment:
Can we keep the `dc-gpios` property uncommented? Populating this property should not switch the MIPI DBI driver to 4 wire mode. Is the issue that the DC gpio would be set to output at boot by the MIPI DBI driver, and that would kick the display into 4 wire mode?

Reply:
Correct, it will not switch to the 4wrie mode.
Default pin connection is for 3wire mode, 4wire mode requires rework on EVKIT. I add it while testing 4wire mode too, then added it as commented to this GPIO can be used for different purpose incase of user would like.
If user would like to use 4wire mode, they need to apply rework and enable this line.

![image](https://github.com/user-attachments/assets/303ad455-8041-4dfa-b15f-addf6315c7e0)

Reply:
Ah, fair reasoning then- thanks for clarifying here